libsql/src/value.rs

```diff
@@ -1,9 +1,9 @@
-use std::str::FromStr;
+use std::{marker::PhantomData, str::FromStr};
```

Comment:
Shouldn't this be imported inside the `serde_` module not outside?

Reply:
ah yes, I didn't compile so I forgot to move this one